includes/abstracts/abstract-wp-job-manager-form.php

```diff
@@ -136,7 +136,17 @@ public function process() {
 	 * @param array $atts Attributes to use in the view handler.
 	 */
 	public function output( $atts = [] ) {
-		$this->enqueue_scripts();
```

Comment:
If I'm understanding correctly, the current `enqueue_scripts()` overrides in the descendant classes (`WP_Job_Manager_Form_Submit_Job`, `WP_Resume_Manager_Form_Submit_Resume`) would solve the problem of detecting if it's a resume or submit job form already, why move this logic up to the parent class?

Reply:
I thought so, too - but in Resumes the class is never instantiated, it's just output through the parent class, so overriding the child class does nothing.

Reply:
Would that be fixable in Resumes? It'd be nice to avoid having to put addon-specific exceptions in the core plugin.

Reply:
Removed the logic here https://github.com/Automattic/WP-Job-Manager/pull/2773/commits/48835aa4054a34a3d4e54714874314582298588c    and added here https://github.com/Automattic/wpjm-addons/pull/472/commits/8926d93fe7bb9b2a4ea881a34d98190e3f714763